e2e/cypress/support/api/preference.d.ts

```diff
@@ -82,5 +82,17 @@ declare namespace Cypress {
          *   cy.apiSaveDirectChannelShowPreference('user-id', 'other-user-id', 'false');
          */
         apiSaveDirectChannelShowPreference(userId: string, otherUserId: string, value: string): Chainable<Response>;
+
+        /**
+         * Save Collapsed Reply Threads preference.
+         * See https://api.mattermost.com/#tag/preferences/paths/~1users~1{user_id}~1preferences/put
+         * @param {string} userId - User ID
+         * @param {string} value - options are 'on' or 'off'
```

Comment:
This is not consistent with the [command argument itself](https://github.com/mattermost/mattermost-webapp/pull/8494/files#diff-8cdf3bb0f2cd0b35fb9ccda9a2eecd7ae0bc9aae7fb55ba16eb67434d5d5c490R244), where it's declared as `crt`. However, I find `value` to be more sound than `crt` IMO.